glean-core/android/src/main/java/mozilla/telemetry/glean/utils/WorkManagerUtils.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.utils
+
+import android.content.Context
+import androidx.work.WorkInfo
+import androidx.work.WorkManager
+import androidx.work.testing.WorkManagerTestInitHelper
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+
+/**
+ * TEST ONLY FUNCTION.
```

Comment:
Does this need a `VisibleForTesting` annotation?

Reply:
Unfortunately, no. While the function ends up triggered only during tests, it's not used directly only in tests. Instead, it's being used in the uploader classes, which are non-test classes.
One thing we could do, though, is make this `internal` only, as it's only meant to be used by our code.